README.md

```diff
@@ -208,7 +208,7 @@ Additionally, it comes with some basic tests for the shipped
 If you want to add these QA tools, execute the following:
 
 ```bash
-$ composer require --dev phpunit/phpunit squizlabs/php_codesniffer zendframework/zend-test
```

Comment:
phpunit/phpunit must be required explicitly as direct dependency. Please revert that change. 
Depending on the package installed by another dependency is dangerous as dependency specifies only its own compatibility constraints.